performance/README.md

```diff
@@ -0,0 +1,114 @@
+# Howso Platform Performance Considerations
+
+## Overview
+
+This guide covers performance considerations for the Howso Platform and client installation. 
+
+## TLDR
+- Use a dedicated Kubernetes cluster for Howso Platform.
+- Increase the size of NATS, Redis, Postgres.
+- Use cluster autoscaling.
+- Run workloads from a large dedicated client machine.
+
+## Cluster Configuration
+
+### Dedicated Cluster
+
+In the Howso Platform, trainees (also known as workers) are created as new (stateful set) pods on demand.  
+- Individual trainees can be very large - it is not uncommon to dedicate tens of cpu resources and even more Gb of memory to a single trainee.
+- The Kubernetes API server is used to create and manage these pods.  The API server is a critical component of the cluster, can come under load, potentially impacting other cluster workloads. 
+
+As such, the profile of the cluster can be demanding and very dynamic.  Therefore it is recommended to run the Howso Platform on a dedicated cluster for the best performance (for the Howso Platform and other workloads).
+
+
+### Dependent Charts
+
+As discussed extensively elsewhere in this guide, Howso Platform relies on several dependent charts.  NATS, Redis, Postgres, Minio.  These charts can be tuned independently of the Howso Platform chart.  The included [manifests](./manifests) provide some basic, but impactful tuning recommendations. 
+
+The out of the box installations of these charts, particularly NATS, Redis and Postgres are small installations.  These should run reasonably well for basic local examples, but for demanding workloads, across large clusters, increasing the allocated resources is very beneficial. 
+
+In rough order of impact:
+
+#### NATS
+
+NATS is used for inter-service communication.  This [NATS manifest](./manifests/nats.yaml) provides a configuration that runs a cluster of 3 JetStream enabled NATS servers - with increased resources.  Check out the [NATS Chart](https://github.com/nats-io/k8s/tree/main/helm/charts/nats) documentation for more information.
+
+#### Redis
```

Comment:
You mention AWS below.  Should our document describe an all-AWS setup, using Elasticache for Redis, RDS for PostgreSQL, and true S3 in place of MinIO?  What would recommended sizing parameters be in this environment?

Reply:
Ideally yes - would be a later update.